discretisedfield/region.py

```diff
@@ -184,43 +344,43 @@ def edges(self):
         (5, 15, 20)
 
         """
-        return dfu.array2tuple(np.abs(np.subtract(self.p1, self.p2)))
+        return np.abs(np.subtract(self.pmin, self.pmax))
 
-    @functools.cached_property
-    def centre(self):
-        r"""Centre point.
+    @property
+    def center(self):
```

Comment:
Should we keep the `centre` property as an alias?

Reply:
It would be good to keep the alias with a deprecation warning as we want to ultimately move towards American English.